pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2022,6 +2120,56 @@ func xlUnpause(domainName string, domainId int) error {
 	return nil
 }
 
+// Wrapper for domain shutdown thru xlShutdown or rktStop
+func wrapDomainShutdown(status types.DomainStatus, force bool) error {
+
+	var err error
+	log.Infof("wrapDomainShutdown force-%v %s %d\n", force, status.DomainName, status.DomainId)
+
+	if status.IsContainer {
+		// Use rkt tool
+		log.Infof("Using rkt tool ... PodUUID - %s\n", status.PodUUID)
+		err = rktStop(status.PodUUID, force)
+	} else {
```

Comment:
I suggest you unconditionally follow with xlShutdown after rktStop

Reply:
Taken care of in latest commit